internal/utils/k8sutil/k8s.go

```diff
@@ -82,5 +82,6 @@ func PortByNumber(pod *v1.Pod, port int32) *v1.ContainerPort {
 
 // StripContainerID returns a pure container id without the runtime scheme://
 func StripContainerID(id string) string {
-	return strings.Replace(strings.Replace(id, "docker://", "", 1), "cri-o://", "", 1)
+	re := regexp.MustCompile(`^[\w_-]+://`)
```

Comment:
Move the `regexp.MustCompile` var to a global variable and reuse it in this function.  That will increase the efficiency of this func by many times.

Reply:
Done